### DISCUSSION TITLE
Cloning a private repo with node.js

### DISCUSSION THREAD

**Discussion by flojo007** [community/community#30100] [Apps, API and Webhooks] [answered]
Hello,
is there a way to clone a private repo with node.js, if yes how would I do that?

Thanks a lot,
flojo

**Comment by jfullstackdev** [comment, marked as answer]
if you are referring to a private project, you need to have the access first, without that, you can't clone it

**Reply by flojo007** [reply]
I want to copy my own private repo

**Reply by jfullstackdev** [reply]
maybe this one 

https://cheatcode.co/tutorials/how-to-clone-and-sync-a-github-repo-via-node-js

**Reply by flojo007** [reply]
That worked thank you!
